Gemfile

```diff
@@ -26,7 +26,7 @@ gem "resque", "~> 1.25.0"
 gem "resque-retry"
 gem "resque-sentry"
 gem "resque-timeout"
-gem "rubocop", "0.29.1"
+gem "rubocop", "~> 0.29"
```

Comment:
If the new config is not backwards compatible I guess we should indicate rubocop version to be 0.34.x instead?